### ISSUE TITLE
Confusion about Odom data

### ISSUE THREAD

**Issue by FelicxFoster** [ethz-asl/lidar_align#33]
Hi, @ZacharyTaylor. Thanks for your  open source
I got a problem about the Odom message we need to load in the function `loadTformFromROSBAG()`.
Does translation field in geometry_msgs/TransformStamped message in ROS mean the GPS position? like the following picture
![odom数据](https://user-images.githubusercontent.com/33318659/114306136-797c4700-9b0d-11eb-8952-b0f159cd447a.png)
but in the example.bag you provided, the translation is too samll,  such as
![example_odom](https://user-images.githubusercontent.com/33318659/114306379-6c138c80-9b0e-11eb-8f71-0f78dcd455a2.png)
I use my data to calibrate and I got the bad result, and different results on one dataset. Is there something wrong with my data? 
Thanks.

**Comment by LarryDong** [comment]
Hi I think the `TransformStamped` should be in "meter", which is 0 at the beginning. Not GPS. You could minus the offset (at the beginning) from you GPS data.

**Comment by FelicxFoster** [comment]
@LarryDong Thanks for your reply. I will try. Could you share your dataset? I want to compare the difference of my dataset, which has the bad result for calibration.

**Comment by LarryDong** [comment]
@FelicxFoster 
Check the data from the author here. [https://github.com/ethz-asl/lidar_align/issues/5#issuecomment-442766071](https://github.com/ethz-asl/lidar_align/issues/5#issuecomment-442766071)

**Comment by FelicxFoster** [comment]
@LarryDong yeah, I have tried the author's data. However, I don't think is a good example, like author said its just he holding the lidar and randomly waving it around. So I hope you could share your data if you have, thanks.

**Comment by LarryDong** [comment]
@FelicxFoster 
Sorry that I do not have a record.
